<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
  <title>Xuran Wang (汪旭冉) | Home</title>
  <meta name="description" content="">

  
  
    <meta property="og:image" content="https://avatars.githubusercontent.com/u/29867659?s=400&u=906f3ca1583e47a92cddbe0a94670a27a8f52b81&v=4" />
    <meta property="og:title" content="Xuran Wang (汪旭冉) | Home" />   
  
  <meta property="og:description" content="" />
  <meta name="author" content="Xuran Wang">
  
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css" integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
  
  <link rel="preload" href="https://fonts.googleapis.com/css?family=Saira+Extra+Condensed:100,200,300,400,500,600,700,800,900&display=swap" as="style" onload="this.onload=null;this.rel='stylesheet'">
    <noscript><link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Saira+Extra+Condensed:100,200,300,400,500,600,700,800,900&display=swap"></noscript>
  <link rel="preload" href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,600i,700,700i,800,800i&display=swap" as="style" onload="this.onload=null;this.rel='stylesheet'">
      <noscript><link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,600i,700,700i,800,800i&display=swap"></noscript>
  <link rel="preload" href="https://use.fontawesome.com/releases/v5.12.1/css/all.css" as="style" onload="this.onload=null;this.rel='stylesheet'">
    <noscript><link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.12.1/css/all.css"></noscript>
  <link rel="preload" href="https://cdnjs.cloudflare.com/ajax/libs/devicons/1.8.0/css/devicons.min.css" integrity="sha512-JW3fT0YTK7pT7w437SoX6GcW76jOZ6E0jGmrqBAcloC4GKT+njHOY4fX5KxJ9WfIXTkNrAF994525fAHp+KCxg==" crossorigin="anonymous" as="style" onload="this.onload=null;this.rel='stylesheet'">
    <noscript><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/devicons/1.8.0/css/devicons.min.css" integrity="sha512-JW3fT0YTK7pT7w437SoX6GcW76jOZ6E0jGmrqBAcloC4GKT+njHOY4fX5KxJ9WfIXTkNrAF994525fAHp+KCxg==" crossorigin="anonymous"></noscript>
  <link rel="preload" href="https://cdnjs.cloudflare.com/ajax/libs/simple-line-icons/2.5.5/css/simple-line-icons.min.css" integrity="sha512-QKC1UZ/ZHNgFzVKSAhV5v5j73eeL9EEN289eKAEFaAjgAiobVAnVv/AGuPbXsKl1dNoel3kNr6PYnSiTzVVBCw==" crossorigin="anonymous" as="style" onload="this.onload=null;this.rel='stylesheet'">
    <noscript><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/simple-line-icons/2.5.5/css/simple-line-icons.min.css" integrity="sha512-QKC1UZ/ZHNgFzVKSAhV5v5j73eeL9EEN289eKAEFaAjgAiobVAnVv/AGuPbXsKl1dNoel3kNr6PYnSiTzVVBCw==" crossorigin="anonymous"></noscript>
  
  <link rel="preload" href="css/resume.css" as="style" onload="this.onload=null;this.rel='stylesheet'">
    <noscript><link rel="stylesheet" href="css/resume.css"></noscript>
  <link rel="preload" href="css/tweaks.css" as="style" onload="this.onload=null;this.rel='stylesheet'">
    <noscript><link rel="stylesheet" href="css/tweaks.css"></noscript>
  <link rel="preload" href="css/resume-override.css" as="style" onload="this.onload=null;this.rel='stylesheet'">
    <noscript><link rel="stylesheet" href="css/resume-override.css"></noscript>
  <meta name="generator" content="Hugo 0.102.0" />
  
   
  
</head>
<body id="page-top">
  <nav class="navbar navbar-expand-lg navbar-dark bg-primary fixed-top" id="sideNav">
  <a class="navbar-brand js-scroll-trigger" href="#page-top">
    <span class="d-block d-lg-none">Xuran Wang</span>
    <span class="d-none d-lg-block">
      <img class="img-fluid img-profile rounded-circle mx-auto mb-2" src="https://avatars.githubusercontent.com/u/29867659?s=400&amp;u=906f3ca1583e47a92cddbe0a94670a27a8f52b81&amp;v=4" alt="">
    </span>
  </a>
  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
    <span class="navbar-toggler-icon"></span>
  </button>
  <div class="collapse navbar-collapse" id="navbarSupportedContent">
    <ul class="navbar-nav">
      <li class="nav-item">
        <a class="nav-link js-scroll-trigger" href="/#about">About</a>
      </li>
      
      
          <li class="nav-item">
            <a class="nav-link js-scroll-trigger" href="/#projects">Projects</a>
          </li>
      
      
      
          <li class="nav-item">
            <a class="nav-link js-scroll-trigger" href="/#Publications">Publications</a>
          </li>
      
      
      
          <li class="nav-item">
            <a class="nav-link js-scroll-trigger" href="/#education">Education</a>
          </li>
      
      
    </ul>
  </div>
</nav>

  <div class="container-fluid p-0">
    
   

      <section class="resume-section p-3 p-lg-5 d-flex d-column" id="about">
        <div class="my-auto">
          <div class="resume-item d-flex flex-column flex-md-row">
            <div class="mr-auto">

              <h1 class="mb-0">Xuran
                <span class="text-primary">Wang</span>
              </h1>
              <div class="subheading mb-5">New York, NY ·
                <a href="mailto:xuran-remove-[dot]wang[at]mssm[dot]edu">xuran<span style="display:none">-remove-</span>[dot]wang[at]mssm[dot]edu</a>
              </div>

            </div>

            

        </div>
        <div>
          <p>Welcome to my Homepage!</p>
<p>I&rsquo;m an Assistant Professor at <a href="https://icahn.mssm.edu/research/seaver">Seaver Autism Center</a> of Research &amp; Treatment at <a href="https://icahn.mssm.edu/">Mount Sinai</a>.</p>
<p>Before joining Mount Sinai, I worked as a postdoctoral researcher with <a href="http://www.stat.cmu.edu/~roeder/">Kathryn Roeder</a> at Carnegie Mellon University. I earned my PhD degree in University of Pennsylvania in 2019. My <a href="./Xuran_Wang_thesis.pdf">thesis</a> &ldquo;Mendelian randomization and single cell deconvolution: Two problems in statistical genetics&rdquo; is advised by <a href="https://statistics.wharton.upenn.edu/profile/nzh/">Nancy Zhang</a>. I also worked with <a href="https://statistics.wharton.upenn.edu/profile/dsmall/">Dylan Small</a> on Mendelian randomization problems and <a href="https://www.med.upenn.edu/apps/faculty/index.php/g275/p8122973">Mingyao Li</a> on statistical modeling of single-cell RNA sequencing.</p>
<p>I received my Bachelor degree of Statistics from University of Science and Technology of China (<a href="https://en.ustc.edu.cn/">USTC</a>), <a href="https://en.scgy.ustc.edu.cn/">School of the Gifted Young</a>.</p>
<p>Here is a current copy of my <a href="./Xuran_Wang_CV_2022.pdf">CV</a>.</p>
<h3 id="research-interests">Research Interests</h3>
<p>My research interests sits in the intersection of statistical genomics, computational biology and neuroscience. I was trained as a statistician and mathematician before diving into the genetics and genomics world. I have broad interests in statistical
genetics and genomics, multiomics integration, gene regulatory networks and spatial transcriptomics.</p>
<blockquote>
<h3 id="opening">Opening</h3>
<p>I am currently looking for postdocs interested in statistical genetics and genomics. Please email a CV to apply.</p>
<p>This is a two year extentable postdoctoral position in Wang&rsquo;s lab. The successful candidate will apply cutting edge methods in statistics and machine learning to solve scientific problems emerging from genetics, using modern large-scale genomics data such as single-cell sequencing data. We are looking for highly motivated individuals with a strong background in statistical methodology, and a genuine interest in science and data-driven research.</p>
<p>Applicants must have (1) a Ph.D. in statistics, biostatistics, computational biology, computer science or other related quantitative field, (2) strong computing skills, (3) good communication skills.</p>
</blockquote>

          <h3>Contact Me</h3>
          <ul class="list-inline list-social-icons mb-0">
            
                <li class="list-inline-item">
                  <a href="https://scholar.google.com/citations?user=GY3j6fUAAAAJ&amp;hl=en" data-toggle="tooltip" title="Google Scholar" data-offset="0,10" rel="me">
                    <span class="fa-stack fa-lg">
                      <i class="fa fa-circle fa-stack-2x"></i>
                      <i class="fab fa-google scholar fa-stack-1x fa-inverse"></i>
                    </span>
                  </a>
                </li>
            
                <li class="list-inline-item">
                  <a href="https://www.linkedin.com/in/xuran-wang-08436a143/" data-toggle="tooltip" title="LinkedIn" data-offset="0,10" rel="me">
                    <span class="fa-stack fa-lg">
                      <i class="fa fa-circle fa-stack-2x"></i>
                      <i class="fab fa-linkedin fa-stack-1x fa-inverse"></i>
                    </span>
                  </a>
                </li>
            
                <li class="list-inline-item">
                  <a href="https://github.com/xuranw/" data-toggle="tooltip" title="GitHub" data-offset="0,10" rel="me">
                    <span class="fa-stack fa-lg">
                      <i class="fa fa-circle fa-stack-2x"></i>
                      <i class="fab fa-github fa-stack-1x fa-inverse"></i>
                    </span>
                  </a>
                </li>
            
                <li class="list-inline-item">
                  <a href="https://twitter.com/xuranw" data-toggle="tooltip" title="Twitter" data-offset="0,10" rel="me">
                    <span class="fa-stack fa-lg">
                      <i class="fa fa-circle fa-stack-2x"></i>
                      <i class="fab fa-twitter fa-stack-1x fa-inverse"></i>
                    </span>
                  </a>
                </li>
            
          </ul>
        </div>
      </div>
    </section>

   
   
      
         
         <section class="resume-section p-3 p-lg-5 d-flex flex-column" id="projects">
  <div class="my-auto" id="projects-content">
    <div class=" pb-2 pt-3">
      <h2><a href="./projects/creations/">Projects</a></h2>
      <!-- raw HTML omitted -->
<!-- raw HTML omitted -->
<!-- raw HTML omitted -->
<!-- raw HTML omitted -->
      
    </div>
  
    
    
      
<h3 class="mb-3"><a href="./projects/creations/loccsn/">Constructing local cell-specific networks from single-cell data</a></h3>
<div class="resume-item d-flex flex-column flex-md-row mb-3">

  <div class="resume-content" style="width:800px;">
    
      <img src="https://raw.githubusercontent.com/xuranw/locCSN/main/Workflow_new.png" style="max-height:300px;max-width:500px;"/>
    
  </div>
  <div class="resume-date">
    <p>Understanding gene regulatory networks is a topic of great interest because it can provide insights into cellular development, and identify factors that differ between normal and abnormal cells and phenotypes. Single-cell RNA sequencing provides a unique opportunity to gain understanding at the cellular level, but the technical features of the data create severe challenges when constructing gene networks. We develop a method that successfully skirts these challenges to estimate a cell-specific network for each single cell and cell type. Application of our algorithm to two brain cell samples furthers our understanding of autism spectrum disorder by examining the evolution of gene networks in fetal brain cells and comparing the networks of cells sampled from case and control subjects.<p>
  </div>
</div>
<div><ul class="tags">
    
      <li><a class="tag" href="/tags/gene-regulatory-network">Gene regulatory network</a></li>
    
      <li><a class="tag" href="/tags/python">Python</a></li>
    
</ul>
</div>

    
      
<h3 class="mb-3"><a href="./projects/creations/music/">Bulk tissue cell type deconvolution with multi-subject single-cell expression reference</a></h3>
<div class="resume-item d-flex flex-column flex-md-row mb-3">

  <div class="resume-content" style="width:800px;">
    
      <img src="https://raw.githubusercontent.com/xuranw/MuSiC/master/FigureMethod.jpg" style="max-height:300px;max-width:500px;"/>
    
  </div>
  <div class="resume-date">
    <p>Knowledge of cell type composition in disease relevant tissues is an important step towards the identification of cellular targets of disease. We present MuSiC, a method that utilizes cell-type specific gene expression from single-cell RNA sequencing (RNA-seq) data to characterize cell type compositions from bulk RNA-seq data in complex tissues. By appropriate weighting of genes showing cross-subject and cross-cell consistency, MuSiC enables the transfer of cell type-specific gene expression information from one dataset to another. When applied to pancreatic islet and whole kidney expression data in human, mouse, and rats, MuSiC outperformed existing methods, especially for tissues with closely related cell types. MuSiC enables the characterization of cellular heterogeneity of complex tissues for understanding of disease mechanisms. As bulk tissue data are more easily accessible than single-cell RNA-seq, MuSiC allows the utilization of the vast amounts of disease relevant bulk tissue RNA-seq data for elucidating cell type contributions in disease.<p>
  </div>
</div>
<div><ul class="tags">
    
      <li><a class="tag" href="/tags/statistical-genomics">statistical genomics</a></li>
    
      <li><a class="tag" href="/tags/deconvolution">deconvolution</a></li>
    
      <li><a class="tag" href="/tags/r">R</a></li>
    
</ul>
</div>

    
      
<h3 class="mb-3"><a href="./projects/creations/sensitivity/">Sensitivity Analysis and Power for Instrumental Variable Studies</a></h3>
<div class="resume-item d-flex flex-column flex-md-row mb-3">

  <div class="resume-content" style="width:800px;">
    
      <img src="https://raw.githubusercontent.com/xuranw/AllelicMR/master/sensitivity.png" style="max-height:300px;max-width:500px;"/>
    
  </div>
  <div class="resume-date">
    <p>In observational studies to estimate treatment effects, unmeasured confounding is often a concern. The instrumental variable (IV) method can control for unmeasured confounding when there is a valid IV. To be a valid IV, a variable needs to be independent of unmeasured confounders and only affect the outcome through affecting the treatment. When applying the IV method, there is often concern that a putative IV is invalid to some degree. We present an approach to sensitivity analysis for the IV method which examines the sensitivity of inferences to violations of IV validity. Specifically, we consider sensitivity when the magnitude of association between the putative IV and the unmeasured confounders and the direct effect of the IV on the outcome are limited in magnitude by a sensitivity parameter. Our approach is based on extending the Anderson–Rubin test and is valid regardless of the strength of the instrument. A power formula for this sensitivity analysis is presented. We illustrate its usage via examples about Mendelian randomization studies and its implications via a comparison of using rare versus common genetic variants as instruments.<p>
  </div>
</div>
<div><ul class="tags">
    
      <li><a class="tag" href="/tags/causal-inference">Causal inference</a></li>
    
      <li><a class="tag" href="/tags/andersonrubin-test">Anderson–Rubin test</a></li>
    
      <li><a class="tag" href="/tags/sensitivity-analysis">Sensitivity analysis</a></li>
    
      <li><a class="tag" href="/tags/r">R</a></li>
    
</ul>
</div>

    
      
<h3 class="mb-3"><a href="./projects/creations/allelicmr/">Allele Specific Information in Mendelian Randomization</a></h3>
<div class="resume-item d-flex flex-column flex-md-row mb-3">

  <div class="resume-content" style="width:800px;">
    
      <img src="https://raw.githubusercontent.com/xuranw/AllelicMR/master/AllelicMR.png" style="max-height:300px;max-width:500px;"/>
    
  </div>
  <div class="resume-date">
    <p>Mendelian randomization (MR) is a method in biomedical research, which uses genotypes as instrument variables to estimate causal effect of exposure to outcome. Conventional estimation is two stage least squares (2SLS). With advanced sequencing technology, allele specific expression information became available. We present a new model to make usage of new information and estimate causal effect using maximum likelihood estimation (MLE). A comparison of precision between 2SLS and MLE is presented with different strength of instrument variables and confounder effect. We also illustrate its usage in finding downstream effect of lncRNA with genotype as instrument.<p>
  </div>
</div>
<div><ul class="tags">
    
      <li><a class="tag" href="/tags/mendelian-randomization">Mendelian randomization</a></li>
    
      <li><a class="tag" href="/tags/r">R</a></li>
    
</ul>
</div>

    
  </div>
  
</section>

      
   
   
   
      
         
         <section class="resume-section p-3 p-lg-5 d-flex flex-column" id="Publications">
  <div class="my-auto" id="projects-content">
    <div class=" pb-2 pt-3">
      <h2><a href="./publications/">Publications</a></h2>
      <!-- raw HTML omitted -->
<!-- raw HTML omitted -->
      
    </div>
  
    
    
      
<div class="resume-item d-flex flex-column flex-md-row mb-5">
  <div class="publication-content mr-auto">
    <h3 class="mb-0"> <a href="./publications/music2/">MuSiC2: cell-type deconvolution for multi-condition bulk RNA-seq data</a></h3>
    <div>Jiaxin Fan, Yafei Lyu, Qihuang Zhang, <strong>Xuran Wang</strong>, Mingyao Li, Rui Xiao</div>
    <p><ul class="tags">
    
</ul>
</p>
  </div>
  <div class="resume-date text-md-right">
    <span class="text-primary">October 2022</span>
    <img src="https://oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/bib/bib_title526563102.svg" style="max-width:250px;padding:5px;"/>
  </div>
</div>

    
      
<div class="resume-item d-flex flex-column flex-md-row mb-5">
  <div class="publication-content mr-auto">
    <h3 class="mb-0"> <a href="./publications/loccsn/">Constructing local cell specific networks from single cell data</a></h3>
    <div><strong>Xuran Wang</strong>, David Choi, Kathryn Roeder</div>
    <p><ul class="tags">
    
</ul>
</p>
  </div>
  <div class="resume-date text-md-right">
    <span class="text-primary">December 2021</span>
    <img src="https://raw.githubusercontent.com/xuranw/locCSN/main/pnas.png" style="max-width:250px;padding:5px;"/>
  </div>
</div>

    
      
<div class="resume-item d-flex flex-column flex-md-row mb-5">
  <div class="publication-content mr-auto">
    <h3 class="mb-0"> <a href="./publications/sceptre/">SCEPTRE improves calibration and sensitivity in single-cell CRISPR screen analysis</a></h3>
    <div>Timothy Barry, <strong>Xuran Wang</strong>, John A Morris, Kathryn Roeder, Eugene Katsevich</div>
    <p><ul class="tags">
    
</ul>
</p>
  </div>
  <div class="resume-date text-md-right">
    <span class="text-primary">December 2021</span>
    <img src="https://raw.githubusercontent.com/xuranw/SCEPTRE/master/genome_biology_logo.png" style="max-width:250px;padding:5px;"/>
  </div>
</div>

    
      
<div class="resume-item d-flex flex-column flex-md-row mb-5">
  <div class="publication-content mr-auto">
    <h3 class="mb-0"> <a href="./publications/ctsallelic/">Detecting cell-type-specific allelic expression imbalance by integrative analysis of bulk and single-cell RNA sequencing data</a></h3>
    <div>Jiaxin Fan, <strong>Xuran Wang</strong>, Rui Xiao, Mingyao Li</div>
    <p><ul class="tags">
    
</ul>
</p>
  </div>
  <div class="resume-date text-md-right">
    <span class="text-primary">March 2021</span>
    <img src="https://raw.githubusercontent.com/xuranw/MuSiC/master/plot_genetics_logo_small.png" style="max-width:250px;padding:5px;"/>
  </div>
</div>

    
      
<div class="resume-item d-flex flex-column flex-md-row mb-5">
  <div class="publication-content mr-auto">
    <h3 class="mb-0"> <a href="./publications/adbrain/">APOE and TREM2 regulate amyloid-responsive microglia in Alzheimer’s disease</a></h3>
    <div>Aivi T Nguyen, Kui Wang, Gang Hu, <strong>Xuran Wang</strong>, Zhen Miao, Joshua A Azevedo, EunRan Suh, Vivianna M Van Deerlin, David Choi, Kathryn Roeder, Mingyao Li, Edward B Lee</div>
    <p><ul class="tags">
    
</ul>
</p>
  </div>
  <div class="resume-date text-md-right">
    <span class="text-primary">October 2020</span>
    <img src="https://raw.githubusercontent.com/xuranw/locCSN/main/Figures/atca_logo.png" style="max-width:250px;padding:5px;"/>
  </div>
</div>

    
      
<div class="resume-item d-flex flex-column flex-md-row mb-5">
  <div class="publication-content mr-auto">
    <h3 class="mb-0"> <a href="./publications/music/">Bulk tissue cell type deconvolution with multi-subject single-cell expression reference</a></h3>
    <div><strong>Xuran Wang</strong>, Jihwan Park, Katalin Susztak, Nancy R Zhang, Mingyao Li.Nature Communications. 10.1 (2019) 1-9.</div>
    <p><ul class="tags">
    
</ul>
</p>
  </div>
  <div class="resume-date text-md-right">
    <span class="text-primary">January 2019</span>
    <img src="https://landportal.org/sites/landportal.org/files/styles/220heightmax/public/Nat-Comm.jpg?itok=eUg0z2rB" style="max-width:250px;padding:5px;"/>
  </div>
</div>

    
      
<div class="resume-item d-flex flex-column flex-md-row mb-5">
  <div class="publication-content mr-auto">
    <h3 class="mb-0"> <a href="./publications/sensitivity/">Sensitivity analysis and power for instrumental variable studies</a></h3>
    <div><strong>Xuran Wang</strong>, Yang Jiang, Nancy R Zhang, Dylan S Small</div>
    <p><ul class="tags">
    
</ul>
</p>
  </div>
  <div class="resume-date text-md-right">
    <span class="text-primary">December 2018</span>
    <img src="https://raw.githubusercontent.com/xuranw/AllelicMR/master/biometrics.png" style="max-width:250px;padding:5px;"/>
  </div>
</div>

    
  </div>
  
</section>

      
   
   
   
               <section class="resume-section p-3 p-lg-5 d-flex flex-column" id="education">
          <div class="my-auto">
            <h2 class="mb-5">Education & Experience</h2>
            
              <div class="resume-item d-flex flex-column flex-md-row mb-5">
                <div class="resume-content mr-auto">
                  <h3 class="mb-0">Icahn School of Medicine at Mount Sinai</h3>
                  <div class="subheading mb-3">Assistant Professor</div>
                  <div>Seaver Autism Center for Research and Treatment</div>
                  <p>Genetics and genomics Science
                </div>
                <div class="resume-date text-md-right">
                  <span class="text-primary">2022 - Current</span>
                </div>
              </div>
              
              <div class="resume-item d-flex flex-column flex-md-row mb-5">
                <div class="resume-content mr-auto">
                  <h3 class="mb-0">Carnegie Mellon University</h3>
                  <div class="subheading mb-3">Postdoctoral Researcher</div>
                  <div>Statistics and Data Science</div>
                  <p>Mentor: Kathryn Roeder
                </div>
                <div class="resume-date text-md-right">
                  <span class="text-primary">2019 -2022</span>
                </div>
              </div>
              
              <div class="resume-item d-flex flex-column flex-md-row mb-5">
                <div class="resume-content mr-auto">
                  <h3 class="mb-0">University of Pennsylvania</h3>
                  <div class="subheading mb-3">Ph.D.</div>
                  <div><a href="https://www.amcs.upenn.edu/">Applied Mathematics and Computational Science</a></div>
                  <p>Thesis Committee: Nancy R. Zhang (Advisor), Mingyao Li, Dylan Small
                </div>
                <div class="resume-date text-md-right">
                  <span class="text-primary">2014 - 2019</span>
                </div>
              </div>
              
              <div class="resume-item d-flex flex-column flex-md-row mb-5">
                <div class="resume-content mr-auto">
                  <h3 class="mb-0"><a href="">University of Science and Technology of China</a></h3>
                  <div class="subheading mb-3">Bachelor of Science</div>
                  <div>Statistics</div>
                  <p>
                </div>
                <div class="resume-date text-md-right">
                  <span class="text-primary">2010 - 2014</span>
                </div>
              </div>
              
           </div>
        </section>

   
   


    <span style="color: #999999; font-size: 60%;">Nifty <a href="https://codepen.io/wbeeftink/pen/dIaDH">tech tag lists</a> from <a class="pen-owner-link" href="https://codepen.io/wbeeftink">Wouter Beeftink</a> </span>
    
  </div>
  
  <script
  src="https://code.jquery.com/jquery-3.6.0.min.js"
  integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4="
  crossorigin="anonymous"></script><script src="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-ho+j7jyWK8fNQe+A12Hb8AhRq26LrZ/JpcUGGOn+Y7RsweNrtN/tE3MoK7ZeZDyx" crossorigin="anonymous"></script>

  
  <script async src="https://cdnjs.cloudflare.com/ajax/libs/jquery-easing/1.4.1/jquery.easing.min.js" integrity="sha512-0QbL0ph8Tc8g5bLhfVzSqxe9GERORsKhIn1IrpxDAgUsbBGz/V7iSav2zzW325XGd1OMLdL4UiqRJj702IeqnQ==" crossorigin="anonymous"></script>
  
  <script async src="/js/resume.js"></script>
  <script async src="https://www.googletagmanager.com/gtag/js?id="></script>
  <script>
    window.dataLayer = window.dataLayer || [];
    function gtag(){dataLayer.push(arguments);}
    gtag('js', new Date());

    gtag('config', '');
  </script>
  

  
</body>
</html>
